lib/menu.rb

```diff
@@ -0,0 +1,50 @@
+require 'csv'
+require 'twilio-ruby'
+require_relative 'order'
+require_relative 'load_menu_module'
+
+class Menu
+  include LoadMenu
+  def initialize(restaurant: "fishnchips.csv", order: Order.new(restaurant: restaurant))
+    @order = order
+    load_menu(restaurant)
+    account_sid = ENV['TWILIOSID']
```

Comment:
Would you break the Twilio function into another class or module to separate it out?